src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -247,6 +248,13 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       System.err.println("Compiler runtime: " + (stop - start) + "ms.");
     }
 
+    try {
+      PrintWriter writer = new PrintWriter(ops.statsfile, "UTF-8");
```

Comment:
WDYT about using `java.nio.Files#write` instead of these three lines?

Reply:
Sounds good to me